### PR TITLE
Add chain-gov_GitHub Repo

### PR DESCRIPTION
**Pull request by nanmuxiusi**

### CODE REVIEW

File: docs/cita/hacking/chain-gov.md [hunk 1]
```diff
@@ -3,4 +3,7 @@ id: chain-gov
 title: 链上治理
 ---
 
-TODO
+更多信息请查看[链上治理 GitHub Repo]
+
+
+[链上治理 GitHub Repo]: https://github.com/cryptape/appchain-gov-general
```
Comment (kaikai1024) [inline docs/cita/hacking/chain-gov.md]: the repo name: `appchain` -> `cita`?

Comment (nanmuxiusi) [reply docs/cita/hacking/chain-gov.md]: > `appchain` -> `cita`?

@rainchen Should we modify the repo title and domain name？Can you fix it?

Comment (rainchen) [reply docs/cita/hacking/chain-gov.md]: ok, let me fix it

Comment (rainchen) [reply docs/cita/hacking/chain-gov.md]: here is the  new repo URL: https://github.com/cryptape/cita-gov-general-demo

**Review by leeyr338 — APPROVED** [review]

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**